File: r/gen_graph.r

install.packages('ggplot2')
library(ggplot2)

data <- read.csv("C:\\Users\\truma\\Documents\\DataVis\\a2-DataVis-5ways\\cars-sample.csv", header=TRUE, sep=",")
print(data)
wt <- data$Weight
mpg <- data$MPG
manu <- data$Manufacturer
p <- ggplot()
p <- p + geom_point(data=data, aes(wt, mpg, alpha=0.5, size=wt, color=manu)) + scale_size_continuous(name="Weight", range=c(1, 10))
p
#  xlab="Weight", alpha=0.5, col=factor(manu) ,ylab="MPG", pch=19)
methods(plot)

File: python/gen_graph.py
import matplotlib.pyplot as plt
import seaborn as sns
import pandas as pd

data = pd.read_csv("../cars-sample.csv")
print(data.columns)
sns.set_style("darkgrid")
sns.scatterplot(x=data.Weight, y=data.MPG, 
    size=data.Weight, hue=data.Manufacturer, alpha=0.5,
    sizes=(15, 200), palette="muted")
plt.savefig("fig.png")
print(data)